### ISSUE TITLE
Current postition zone (blue circle) shrink with zoom level

### ISSUE THREAD

**Issue by hieutvan** [osmandapp/Osmand#6353]
SW info: OsmAnd~ 3.2.7 on LineageOS Oreo
Steps to reproduce:
* Detect current location on map (must be low precision i.e. indoor or GPS-disabled)
* Zoom the map in, and the blue circle will shrink according to the current zoom level.

**Comment by naoliv** [comment]
Like https://github.com/osmandapp/Osmand/issues/3416#issuecomment-282573067?

**Comment by vshcherb** [comment]
Yes this is duplicate of #3416 and we don't plan to change it for now.
